website_snippet_anchor/static/src/js/anchor_option.js

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2015, 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+"use strict";
+odoo.define("website_snippet_anchor.anchor_option", function (require) {
+    var core = require("web.core"),
+        options = require("web_editor.snippets.options"),
+        utils = require("website.utils"),
+        sprintf = _.str.sprintf,
+        _t = core._t;
+
+    // Option to have anchors in snippets
+    options.registry.anchor = options.Class.extend({
+
+        anchor_ask: function(type) {
+            if(type !== false) {
+                return;
```

Comment:
Since the good result is a deferred, it makes more sense to `return $.Deferred().reject()` when failing. Leave as it was please.